samples/CommunityToolkit.Maui.Sample/Pages/Behaviors/SelectAllTextBehaviorPage.xaml

```diff
@@ -26,20 +26,20 @@
 
 
             <Label FontAttributes="Italic" Text="Entry without the effect, when focused no text will be selected" />
-            <Border BackgroundColor="{StaticResource SoftBorderBackgroundColor}" >
+            <Border BackgroundColor="{StaticResource SoftFrameBackgroundColor}" >
```

Comment:
I would prefer to rename the static resource to SoftBorderBackgroundColor.
1. It better describes, that it applies to border
2. Frame is old and obsolete in .NET MAUI (it even didn't work). We'd like to remove them all and replace them with Border.

Reply:
Thanks @VladislavAntonyuk for the feedback. The static resource has been updated and also I confirmed the SelectAllText page is using Border instead of Frame.